boefjes/boefjes/plugins/kat_testssl_sh_ciphers/main.py

```diff
@@ -26,13 +27,14 @@ def run(boefje_meta: BoefjeMeta) -> list[tuple[set, bytes | str]]:
         detach=True,
     )
 
-    container.wait()
-
-    output = get_file_from_container(container, "tmp/output.json")
-
     try:
+        container.wait(timeout=300)
+        output = get_file_from_container(container, "tmp/output.json")
+    except (docker.errors.DockerException, RequestException) as e:
+        logging.warning("DockerException occurred: %s", e)
+        container.stop()
+        output = ""
```

Comment:
@underdarknl Wouldn't we still want some kind of boefje failure (and possibly reschedule)? I believe testssl would still produce some JSON output even if nothing interesting comes out. I'd rather see something than silently ignoring this because this will still show as a successful boefje run.

Reply:
Testssl doesn't produce an output unfortunately. It hangs before doing anything, see: https://github.com/minvws/nl-kat-coordination/issues/1913#issuecomment-1765991030

Reply:
We can choose to let the boefje fail! That would make sense indeed. Would also mean that I can remove the changes on the normalizer

Reply:
We'd need to differentiate between a timeout in reaching the target (which would mean we need to reschedule), and a hanging process because testssl is unhappy about something else (like trying ssl tests on a non ssl port). The later could just be shortcutted by having the boefje return a succes state but not do any work like we do for other boefjes that do their own work filtering.